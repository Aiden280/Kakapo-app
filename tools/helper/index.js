import fs from 'fs-extra';

// Upload stats.json to `http://webpack.github.io/analyse` for analysis.
// Also you can analyze size with `https://github.com/robertknight/webpack-bundle-size-analyzer`:
// `cat stats.json | analyze-bundle-size`
export const writeStats = stats =>
  fs.writeFile('stats.json', JSON.stringify(stats.toJson(), null, 4));

export default {};
