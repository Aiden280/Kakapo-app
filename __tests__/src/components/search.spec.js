import React from 'react';
import { ImportSearch as Search } from 'components/';
import { getData, createComponentWithIntl } from '../helper';

test('<Search/> render', () => {
  const props = {
    location: { pathname: '/youtube' },
    ...getData('search'),
  };
  const tree = createComponentWithIntl(<Search {...props} />).toJSON();
  expect(tree).toMatchSnapshot();
});
