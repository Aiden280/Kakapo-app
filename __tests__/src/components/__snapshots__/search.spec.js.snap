// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Search/> render 1`] = `
<div
  className="youtube"
>
  <div
    className="group"
  >
    <input
      autoComplete="off"
      defaultValue={undefined}
      name="searchInput"
      onBlur={[Function]}
      onChange={[Function]}
      onFocus={[Function]}
      required={true}
      type="text"
    />
    <label
      htmlFor="searchInput"
    >
      Search YouTube (or enter a YouTube Link)
    </label>
    <span
      className="highlight"
    />
    <span
      className="bar"
    />
  </div>
  <div
    className="youtube-items"
  />
</div>
`;
